### PR TITLE
[libc++][NFC] Status page: fix minor formatting issues

### PR DESCRIPTION
**Pull request by H-G-Hristov**
Fixes several minor formatting issues and omissions I introduced on the status pages in previous patches.

### CODE REVIEW

File: libcxx/docs/ReleaseNotes/19.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Implemented Papers
 - P2637R3 - Member ``visit``
 - P2652R2 - Disallow User Specialization of ``allocator_traits``
 - P2819R2 - Add ``tuple`` protocol to ``complex``
-- P2495R3 - Interfacing ``stringstream``s with ``string_view``
+- P2495R3 - Interfacing ``stringstreams`` with ``string_view``
```
Comment (frederick-vs-ja) [inline libcxx/docs/ReleaseNotes/19.rst:44]:
```suggestion
- P2495R3 - Interfacing ``stringstream``\ s with ``string_view``
```

File: libcxx/docs/Status/Cxx2cPapers.csv [hunk 2]
```diff
@@ -6,7 +6,7 @@
 "`P2545R4 <https://wg21.link/P2545R4>`__","LWG","Read-Copy Update (RCU)","Varna June 2023","","",""
 "`P2530R3 <https://wg21.link/P2530R3>`__","LWG","Hazard Pointers for C++26","Varna June 2023","","",""
 "`P2538R1 <https://wg21.link/P2538R1>`__","LWG","ADL-proof ``std::projected``","Varna June 2023","|Complete|","18.0","|ranges|"
-"`P2495R3 <https://wg21.link/P2495R3>`__","LWG","Interfacing ``stringstream``s with ``string_view``","Varna June 2023","|Complete|","19.0",""
+"`P2495R3 <https://wg21.link/P2495R3>`__","LWG","Interfacing ``stringstreams`` with ``string_view``","Varna June 2023","|Complete|","19.0",""
```
Comment (frederick-vs-ja) [inline libcxx/docs/Status/Cxx2cPapers.csv:9]:
```suggestion
"`P2495R3 <https://wg21.link/P2495R3>`__","LWG","Interfacing ``stringstream``\ s with ``string_view``","Varna June 2023","|Complete|","19.0",""
```

**Review by frederick-vs-ja — COMMENTED** [review]
Per https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html escaped space `\ ` should work.

File: libcxx/docs/ReleaseNotes/19.rst (hunk 1 above)
Comment (H-G-Hristov) [reply libcxx/docs/ReleaseNotes/19.rst:44]: Thank you a lot! I wonder why there isn't an issue here:
https://github.com/llvm/llvm-project/blob/762f762504967efbe159db5c737154b989afc9bb/libcxx/docs/Status/SpaceshipPapers.csv?plain=1#L3

Comment (H-G-Hristov) [reply libcxx/docs/ReleaseNotes/19.rst]: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#character-level-inline-markup-1
The white space shouldn't be necessary. This should be enough:
> \``stringstream\``\s

**Pull request merged**